cmds/portmaster-start/logs.go

```diff
@@ -96,7 +96,7 @@ func getPmStartLogFile(ext string) *os.File {
 	}, info.Version(), ext)
 }
 
-//nolint:deadcode,unused // false positive on linux, currently used by windows only
+//nolint:unused // false positive on linux, currently used by windows only
 func logControlError(cErr error) {
 	// check if error present
 	if cErr == nil {
```

Comment:
we cloud move it to a `_windows.go` file then

Reply:
Good point. I'll need to have to look into that more and understand it again. ;)
Will add a TODO tough.